sockets/sockets.go

```diff
@@ -7,6 +7,8 @@ import (
 type ContainerInfo struct {
 	ID                string
 	DockerEnvironment []string
+	HostName          string
+	HostIP            net.IP
 }
```

Comment:
For parsing this information in StackState it would be nice to know which fields can be null/empty. So e.g. hostIP can be nil/empty i saw, what about the rest?